addon/components/ilios-calendar-month.js

```diff
@@ -23,6 +24,18 @@ export default class IliosCalendarMonthComponent extends Component {
     });
   }
 
+  get singleDayEvents() {
+    return this.nonIlmPreWorkEvents.filter((event) =>
+      moment(event.startDate).isSame(moment(event.endDate), 'day')
```

Comment:
Can you swap these with [luxon](https://moment.github.io/luxon/#/math?id=comparing-datetimes)? I'd like to push to get rid of moment at some point this year. Example of importing and using it is in [session-object](https://github.com/ilios/common/blob/96e9981a1a9a7d6d80a13d1890fc64e9d0f5ec40/addon/classes/session-object.js#L7)

Reply:
:heavy_check_mark: